### PR TITLE
🐞[Bug] version label was incorrectly formatted.

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Whats wrong?
User support ticket showed the bug.

Must be squashed

### CODE REVIEW

File: loafwallet/src/Constants/Constants.swift [hunk 1]
```diff
@@ -261,5 +261,5 @@ struct C {
 struct AppVersion {
 	static let buildNumber = Bundle.main.infoDictionary?["CFBundleVersion"] as? String
 	static let versionNumber = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
-	static let string = "v." + versionNumber! + " (\(buildNumber!))"
+	static let string = "v" + versionNumber! + " (\(buildNumber!))"
```
Comment (josikie) [inline loafwallet/src/Constants/Constants.swift:264]: Nice! @kcw-grunt

**Review by josikie — APPROVED** [review]
Nice! @kcw-grunt

**Comment by kcw-grunt** [comment]
Thanks @josikie . Next final step is to approve the pr. (It's a condition of passing the PR.

**Comment by josikie** [comment]
I approved the pr @kcw-grunt , thank you!

**Pull request merged**